### ISSUE TITLE
Unable to get property 'toFixed' of undefined or null reference in ng-animate.js method onAnimationProgress(event).....

### ISSUE THREAD

**Issue by erfxyz** [angular/angular.js#13182]
When testing a localhost version of "http://dwmkerr.github.io/angular-modal-service/" which is in the attached ZIP file setup in Visual Studio 2013
    (localhost URL is http://localhost:51721/AngularModalServiceSamples2.html), I encountered the error only in IE9 for ng-Animate method "onAnimationProgress(event)". Attached is the screen capture image and the ZIP file to reproduce the problem in localhost using Visual Studio 2013.
![nganimate_issue_tofixed](https://cloud.githubusercontent.com/assets/15347878/10767729/f220ca9c-7cb3-11e5-96a0-e51bf945a04a.png)

**Comment by erfxyz** [comment]
I have the ZIP file to reproduce the problem but could not upload the Project zip file as Github does not allow me to upload the zip file.

**Comment by Narretz** [comment]
Do you think you could reproduce the problem in a minimal demo in a plnkr etc? It's very cumbersome to run a big project locally.

**Comment by erfxyz** [comment]
I am trying to inline the code and test it. I will get back to you once I reproduced it.

Edwin
x738177

**Comment by erfxyz** [comment]
It is in Plunker : http://plnkr.co/edit/NTd0MG1YoLoEqQywxyo2?p=preview

However, when in Plunker it does not reproduce the error, only when I
launch the page in Visual Studio 2013 running in localhost do I get the error.

Edwin
x738177

From: Fajardo, Edwin
Sent: Tuesday, October 27, 2015 3:29 PM
To: 'angular/angular.js'; 'angular/angular.js'
Subject: RE: {NW} Re: [angular.js] Unable to get property 'toFixed' of undefined or null reference in ng-animate.js method onAnimationProgress(event)..... (#13182)

I am trying to inline the code and test it. I will get back to you once I reproduced it.

Edwin
x738177

**Comment by erfxyz** [comment]
Issue #13182 is the same as Github Issue #10387. Plunker URL is
      http://plnkr.co/edit/NTd0MG1YoLoEqQywxyo2?p=preview

but the HTML and template html’s need to run in Visual Studio 2013
as localhost to reproduce the problem as it only occurs in Visual Studio 2013.

Edwin
x738177

**Comment by Narretz** [comment]
If it's the same as #10387, we should continue the discussion there.

**Comment by erfxyz** [comment]
I was trying to add to 10387 but only allow me to create a new entry.

Edwin
x738177

**Comment by Narretz** [comment]
@erfxyz You cannot comment on the other issue?

**Comment by erfxyz** [comment]
Yes, I cannot even add any comment and your link forced me to create a new issue.

Also, I tried installing through nugget Angular UI Bootstrap on the same Visual Studio 2013
solution, the problem went away, so this might be that nugget also updated the Angular JS
and Bootstrap components. I tested the same HTML page in our Project Visual Studio 2013
last night and the problem is NOT  OCCURRING anymore.

I have just arrived at the office this morning, tested the same page today and it is now
working perfectly in localhost Visual Studio 2013.

Edwin
x738177
